CMakeLists.txt

```diff
@@ -213,6 +213,10 @@ option(CMAKE_INSTALL_RPATH_USE_LINK_PATH "build with automatic rpath" TRUE)
 
 if(APPLE)
     set(CMAKE_MACOSX_RPATH ON)
+
+    if(CURRENT_OSX_VERSION GREATER_EQUAL 13.0)
+        set(CMAKE_EXE_LINKER_FLAGS "-Wl,-rpath,/usr/local/lib")
```

Comment:
I don't have OSX to verify, but hardcoding this rpath would cause issues with running the development binaries. i.e. trying to run executable from build directory, it would load the installed libraries instead of libraries from build directory.

this issue should be handled by using https://cmake.org/cmake/help/latest/prop_tgt/INSTALL_RPATH.html